docs/introduction.rst

```diff
@@ -1,142 +1,132 @@
-  to innovate their distribution.
-- **Stability**: breakage in a Flatpak application will not risk the system from breaking.
-  This is because Flatpak applications and runtimes are contained to not interfere
-  with the system altogether.
-- **Rootless install**: elevated privileges are not required when installing a Flatpak
-  application or a runtime.
+  distribution, including non-GNU distributions, systemd-free distributions,
+  immutable distributions and various architectures without requiring the
+  developer to have access to the relevant hardware.
+- **Space for innovation**: Flatpak allows distribution maintainers to focus on
+  innovating their distribution without being burdened by packaging concerns.
+- **Stability**: breakages in Flatpak applications do not affect the system
+  since they run in isolated environments.
+- **Rootless install**: Flatpak does not require elevated privileges to install
+  applications or runtimes.
 - **Sandboxed applications**: one of Flatpak's main goals is to increase the security of desktop
-  systems by isolating applications from one another. This is achieved using sandboxing and means
-  that, by default, applications that are run with Flatpak have limited access to the host environment.
+  systems. This is achieved by isolating applications from one another and limiting their
+  access to the host environment.
 
-Flatpak has some major advantages over other universal approaches to distributing
-applications on Linux:
+Flatpak also offers advantages over other universal approaches to Linux application distribution:
 
-- **Decentralized by design**: while Flatpak does provide a centralized service for distributing
-  applications, it also allows decentralized hosting and distribution, so that
-  application developers or downstreams can host their own applications and
+- **Decentralized by design**: Flatpak offers decentralized hosting and distribution,
+  allowing developers or downstreams to host their own applications and
   application repositories.
-- **Desktop integration**: Flatpak also offers native integration for the main Linux desktops, so that
-  users can easily browse, install, run and use Flatpak applications through
-  their existing desktop environment and tools.
-- **Space efficiency**: Flatpak deduplicates libraries and other files used by multiple
-  applications to save megabytes or even gigabytes worth of storage depending on
-  the amount of applications installed.
-- **Delta updates**: only changed files are downloaded for updates.
+- **Desktop integration**: Flatpak offers native integration with many Linux desktop environments, allowing
+  users to easily browse, install, run, and manage Flatpak applications.
+- **Space efficiency**: Flatpak saves storage by deduplicating libraries and other files used by multiple
+  applications.
+- **Delta updates**: only the differences between versions are downloaded during updates.
```

Comment:
the previous wording was fine, `only modified files are downloaded`.

Reply:
difference between versions is a bit unclear.

Reply:
I read somewhere that it's not on a file-by-file basis and that only the actual binary differences are downloaded. Saying `only modified files are downloaded` may give the impression that app developers should split up their programs as much as possible into smaller files for efficient updates.

Reply:
There is an advantage to splitting things up into smaller updates yes. Maybe use `only updated data is downloaded`